statsmodels/discrete/tests/test_discrete.py

```diff
@@ -1772,6 +1773,244 @@ def test_predict_prob(self):
         assert_allclose(chi2[:], (0.64628806058715882, 0.98578597726324468),
                         rtol=0.01)
 
+class TestNegativeBinomialPNB2Newton(CheckModelResults):
+    @classmethod
+    def setup_class(cls):
+        data = sm.datasets.randhie.load()
+        exog = sm.add_constant(data.exog, prepend=False)
+        cls.res1 = NegativeBinomialP(data.endog, exog, p=2).fit(method='newton', disp=0)
+        res2 = RandHIE()
+        res2.negativebinomial_nb2_bfgs()
+        cls.res2 = res2
+
+    #NOTE: The bse is much closer precitions to stata
+    def test_bse(self):
+        assert_almost_equal(self.res1.bse, self.res2.bse, DECIMAL_3)
```

Comment:
in general: use `assert_allclose` with appropriate choice of atol and/or rtol.
`assert_almost_equal` is much less flexible and comes from code that was written before numpy had `assert_allclose`